js/layout.js

```diff
@@ -501,7 +500,7 @@ class ReferencesBrowser extends HTMLElement {
                     }
                 }
                 else{
-                    witnessList.querySelector("li").innerHTML = `<b> No Witnesses Found.  Create one now! </b>`    
+                    witnessList.querySelector("li").innerHTML = `<b> No Witness References Found.  Add One Above! </b>`    
```

Comment:
This feels like a guard clause and we don't need this `else`

Reply:
I am ok with that change if you want to do it.  Just test it and make sure the right message makes it to the interface.